tests/cargo-kani/ws-specified/subcrate3/src/lib.rs

```diff
@@ -0,0 +1,10 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+//! One of 3 sub packages used to test specifying packages with -p
+//! flag.
+
+#[kani::proof]
+fn check_libcrate_proof() {
```

Comment:
Should this be `check_subcrate3_proof`? Also, do you want to make this test failing to make sure the `expected` test excludes it?